RevPiDevice.c

```diff
@@ -84,6 +84,17 @@ const MODGATECOM_IDResp RevPiConnect_ID_g = {
 	.i16uFeatureDescriptor = MODGATE_feature_IODataExchange
 };
 
+const MODGATECOM_IDResp RevPiConnect4_ID_g = {
+	.i32uSerialnumber = 1,	 // TODO: Read from HAT EEPROM ?
+	.i16uModulType = KUNBUS_FW_DESCR_TYP_PI_CONNECT_4,
+	.i16uHW_Revision = 1,
+	.i16uSW_Major = 1,
+	.i16uSW_Minor = 0,
+	.i32uSVN_Revision = 0,
+	.i16uFBS_InputLength = 6,
+	.i16uFBS_OutputLength = 5,
+	.i16uFeatureDescriptor = MODGATE_feature_IODataExchange
```

Comment:
This needs to be set to 0.

`MODGATE_feature_IODataExchange` is defined in <a href="https://github.com/RevolutionPi/piControl/blob/1985f74eba9ebedaa310170945bc0c2e712ad7a7/ModGateComMain.h#L128">ModGateComMain.h</a> as "supports data-exchange using ethernet (e.g. mGate)" and the Connect 4 isn't capable of that.

Changing this doesn't really make a difference, it's just for the sake of correctness. The same erroneous value was copy-pasted to the Compact and Flat entries before.

Reply:
Fixed: https://github.com/RevolutionPi/piControl/pull/94/commits/744244e9f0e32878c7797968f26b20ffd14da2e3#diff-b4a4da48759781523712bc80ee2037e23044aa9cdc6bfbf8cfcad35ca0b5522aR96